src/test/java/uk/co/orchardsystems/cinema_ticketing/CinemaServiceTest.java

```diff
@@ -0,0 +1,110 @@
+        String expected = "[ { 'rating': '15', 'id': 1, 'title': 'Blade Runner' }, { 'rating': '18', 'id': 2, 'title': 'Alien' }, { 'rating': '12', 'id': 3, 'title': 'The Lord of the Rings: The Fellowship of the Ring' }, { 'rating': '12', 'id': 5, 'title': 'Lawrence of Arabia' }, { 'rating': '18', 'id': 6, 'title': 'The Thing' } ]";
+        JSONAssert.assertEquals(expected, cinemaService.getMoviesList(1), JSONCompareMode.LENIENT);
+        expected = "[ { 'rating': '15', 'id': 1, 'title': 'Blade Runner' }, { 'rating': '18', 'id': 2, 'title': 'Alien' }, { 'rating': '12', 'id': 3, 'title': 'The Lord of the Rings: The Fellowship of the Ring' }, { 'rating': '15', 'id': 4, 'title': 'The Big Lebowski' }, { 'rating': '12', 'id': 5, 'title': 'Lawrence of Arabia' }, { 'rating': '18', 'id': 6, 'title': 'The Thing' } ]";
+        JSONAssert.assertEquals(expected, cinemaService.getMoviesList(2), JSONCompareMode.LENIENT);
+    }
+
+    @Test
+    public void whenWrongCinemaIdIsProvidedForGetMoviesList_thenCorrectExceptionIsThrown() {
+        assertThrows(JSONException.class, () -> {
+            cinemaService.getMoviesList(-1);
+        });
+    }
+
+    @Test
+    public void whenCinemaIdAndMovieIdIsProvided_thenReturnedScreeningsAreCorrect() throws JSONException, NoScreeningsFoundException {
+        String expected = "[ { 'id': 1, 'date': '7/8/2018', 'screenId': 1, 'time': '9:00' }, { 'id': 4, 'date': '7/8/2018', 'screenId': 1, 'time': '22:45' }, { 'id': 5, 'date': '7/8/2018', 'screenId': 2, 'time': '10:00' } ]";
+        JSONAssert.assertEquals(expected, cinemaService.getScreeningsForMovie(1, 1), JSONCompareMode.LENIENT);
+        expected = "[ { 'id': 7, 'date': '7/8/2018', 'screenId': 2, 'time': '19:15' } ]";
+        JSONAssert.assertEquals(expected, cinemaService.getScreeningsForMovie(1, 5), JSONCompareMode.LENIENT);
+        expected = "[ { 'id': 1, 'date': '7/8/2018', 'screenId': 1, 'time': '9:00' }, { 'date': '7/8/2018', 'screenId': 2, 'id': 5, 'time': '10:00' } ]";
+        JSONAssert.assertEquals(expected, cinemaService.getScreeningsForMovie(2, 1), JSONCompareMode.LENIENT);
+        expected = "[ { 'id': 8, 'date': '7/8/2018', 'screenId': 2, 'time': '23:45' } ]";
+        JSONAssert.assertEquals(expected, cinemaService.getScreeningsForMovie(2, 6), JSONCompareMode.LENIENT);
+    }
+
+    @Test
+    public void whenWrongCinemaIdISProvidedForGetScreeningsForMovie_thenCorrectExceptionIsThrown(){
+                assertThrows(JSONException.class, () -> {
+            cinemaService.getScreeningsForMovie(-1, 1);
+        });
+    }
+
+        @Test
+    public void whenWrongMovieIdISProvidedForGetScreeningsForMovie_thenCorrectExceptionIsThrown(){
+                assertThrows(NoScreeningsFoundException.class, () -> {
+            cinemaService.getScreeningsForMovie(1, -1);
+        });
+    }
+
+    private static String readFile(String path, Charset encoding) throws IOException {
```

Comment:
Cool, in reality, I wouldn't necessarily write a test for dev only data seeding.